fxgl-animation/src/main/kotlin/com/almasb/fxgl/animation/AnimatedValue.kt

```diff
@@ -102,32 +102,25 @@ class AnimatedCubicBezierPoint2D
     }
 }
 
-class AnimatedQuadBezierPoint3D
+internal class AnimatedQuadBezierPoint3D
 (val path: QuadCurve) : AnimatedValue<Point3D>(Point3D.ZERO, Point3D.ZERO) {
+	
+	private val Animated2D = AnimatedQuadBezierPoint2D(path)
```

Comment:
Something odd with formatting, please check that spaces are set to 4 and no tabs.